keyboards/kyria/keymaps/winterNebs/keymap.c

```diff
@@ -0,0 +1,743 @@
+/* Copyright 2019 Thomas Baart <thomas@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#ifdef CONSOLE_ENABLE
+	#include <print.h>
+#endif
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  WORKMAN,
+  LOWER,
+  RAISE,
```

Comment:
You don't need these here if they're just aliases for native keycodes (defined on Lines 38 and 39).

```suggestion
```